app/views/vet/confirmation.njk

```diff
@@ -11,13 +11,13 @@
         html: "Booking reference number<br><strong>" + reference + "</strong>"
       }) }}
 
-      <p>The record has been submitted.</p>
       <h2 class="govuk-heading-m">What happens next</h2>
-      <p class="govuk-body">We’ve contacted the farmer to ask them to check and confirm the record. They can claim their funding when they have confirmed this.</p>
+      <p class="govuk-body">We’ve contacted the farmer to tell them they can claim their funding.</p>
       <p class="govuk-body">Invoice the farmer if you have not already done this.</p>
       <h2 class="govuk-heading-m">Contact the Rural Payments Agency</h2>
       <p class="govuk-body">If you have a question about your submission, call the Rural Payments Agency on 03000 200 301</p>
```

Comment:
A full stop needs adding to the end of the sentence.

Reply:
I can't see any change made to this.

Reply:
its already there.